src/ceph.in

```diff
@@ -545,11 +545,24 @@ def main():
             # for both:
             childargs = childargs[2:]
         else:
-            print >> sys.stderr, 'daemon requires at least 3 arguments'
+            print >> sys.stderr, '{0} requires at least {1} arguments'.format(
+                childargs[0], require_args)
             return errno.EINVAL
 
     if sockpath and daemon_perf:
-        DaemonWatcher(sockpath).run()
+        interval = 1
+        count = None
+        if len(childargs) > 0:
+            if not childargs[0].isdigit():
+                print >> sys.stderr, 'daemonperf: interval should be a positive integer'
```

Comment:
Should validate that interval is positive real number, doesn't have to be an integer